Bulk items creation for Knitting

Vigneshsekar opened the pull request.

Vigneshsekar committed:
Add index to ipd item mapping

Vigneshsekar committed:
Create knitting variants and BoM

karthikeyan5 commented on apparelo/apparelo/doctype/knitting/knitting.py:
```diff
@@ -5,11 +5,73 @@
+					bom.submit()
+					boms.append(bom.name)
+				else:
+					boms.append(existing_bom)
```
need to check if the resultant BOM is the same...

karthikeyan5 commented on apparelo/apparelo/doctype/knitting/knitting.py:
```diff
@@ -5,11 +5,73 @@
 	def on_submit(self):
 		create_item_template()
 
+def create_knittng_variants(input_items, knitting_doc):
```
All these functions need to go into the knitting class...

karthikeyan5 commented on apparelo/apparelo/doctype/knitting/knitting.py:
```diff
@@ -131,7 +193,9 @@ def create_item_template():
-	item.submit()
+
+	dia = frappe.get_doc('Item Attribute', 'Dia')
+	print(dia.name)
```
need to remove print statements...

karthikeyan5 committed:
Move Knitting function into the class

> made IPD Mapping Doctypes as Non submitable doctypes - inspired from "GL Entry" doctype.
